plugins/content/pagenavigation/tmpl/default.php

```diff
@@ -10,12 +10,11 @@
 
 defined('_JEXEC') or die;
 
-use Joomla\CMS\Factory;
 use Joomla\CMS\Language\Text;
 use Joomla\CMS\Router\Route;
 
-$lang = Factory::getLanguage(); ?>
-
+$lang = $this->getApplication()->getLanguage();
```

Comment:
Can you revert this one as I would like to do it as part of https://github.com/joomla/joomla-cms/pull/40614.